### PR TITLE
for generation via buildcell, let prep stage return a string and only optionally write to file

### PR DESCRIPTION
**Pull request by simonwengert**
see Issue #53

### CODE REVIEW

**Comment by bernstei** [comment]
I'd actually suggest getting rid of the filename, and let whatever calls it write the string to a file if they want. That's been on my list of things to do for the refactors gap-rss anyway.

**Comment by bernstei** [comment]
Change looks good, but now I see that there's no docstring.  As long as this is in your repo I can't write one, so you'll have to, or we could create a branch in the local repo that's cloned from yours.

**Comment by simonwengert** [comment]
Right, also noted that. I added a doc-string now.

File: wfl/generate/buildcell.py [hunk 1]
```diff
@@ -16,6 +16,47 @@
 # and bond len as values
 def create_input(z, vol_per_atom, bond_lengths, composition=None, RSS_min_vol_factor=0.5,
                  vol_range=(0.95, 1.05), min_sep_factor=0.9, symmops='1-8', natom=(6, 24), odd=None, verbose=False):
+    """
+    Create an input string suitable for a random structure search (RSS) via AIRSS' `buildcell`.
```
Comment (bernstei) [inline wfl/generate/buildcell.py:20]: sphinx-apidoc works better with two backquotes, so \`\`buildcell\`\`

File: wfl/generate/buildcell.py [hunk 2]
```diff
@@ -16,6 +16,47 @@
...
+        between atoms.
+    symmops: str, default '1-8'
+        Build structures have a specified number of symmetry operations.
+        For crystals, the allowed values are (1,2,3,4,6,8,12,16,24,48).
+        For clusters (indicated with #CLUSTER), the allowed values are
+        (1,2,3,5,4,6,7,8,9,10,11,12,24). Ranges are allowed (see default).
+    natom: tupe(2), default (6, 24)
+        Defines the range of total numbers of atoms to be present
+        in build structures.
+    odd: str, default None
```
Comment (bernstei) [inline wfl/generate/buildcell.py:50]: There's also an `"also"` value.  I suggest
```
odd: str, default None
    control odd/even numbers of atoms in cell. Use ``"only"`` to only produce cells with 
    odd numbers of atoms, ``"also"`` to produce both odd and even, and ``None`` for even only.
```

**Comment by bernstei** [comment]
Thanks @simonwengert .  Looks good, except for some tiny wording changes (see in-code comments above).  It'd be nice to have a test, I guess added to `tests/test_buildcell.py`.  If you feel like doing that, great, if not, we should open an issue.

**Comment by bernstei** [comment]
Looks good now.  Just let me know if I should merge now, or if you're working on a test.

**Comment by simonwengert** [comment]
@bernstei I'm writing on a test. Actually, I've just finished writing it and will push it now.

**Comment by bernstei** [comment]
Fancy.  Yes, that looks good.  @simonwengert if you think it's ready, I'll merge.  If you think it needs another look, let me know.

**Comment by simonwengert** [comment]
I would think it's ready to be merged

**Pull request merged**